boards/arm/nucleo_f446ze/doc/index.rst

```diff
@@ -0,0 +1,243 @@
+
+- `STM32F446ZE on www.st.com`_
+- `STM32F446 reference manual`_
+- `STM32F446 Datasheet`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_f446ze board configuration supports the following hardware features:
+
++------------+------------+-------------------------------------+
+| Interface  | Controller | Driver/Component                    |
++============+============+=====================================+
+| NVIC       | on-chip    | nested vector interrupt controller  |
++------------+------------+-------------------------------------+
+| UART       | on-chip    | serial port                         |
++------------+------------+-------------------------------------+
+| PINMUX     | on-chip    | pinmux                              |
++------------+------------+-------------------------------------+
+| GPIO       | on-chip    | gpio                                |
++------------+------------+-------------------------------------+
+| PWM        | on-chip    | pwm                                 |
++------------+------------+-------------------------------------+
+| I2C        | on-chip    | i2c                                 |
++------------+------------+-------------------------------------+
+| I2S        | on-chip    | i2s                                 |
++------------+------------+-------------------------------------+
+| SPI        | on-chip    | spi                                 |
++------------+------------+-------------------------------------+
+| ADC        | on-chip    | adc                                 |
++------------+------------+-------------------------------------+
+| USB OTG FS | on-chip    | usb_device                          |
++------------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_f446ze/nucleo_f446ze_defconfig``
+
+
+Connections and IOs
+===================
+
+Nucleo F446ZE Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+Available pins:
+---------------
+.. image:: img/nucleo_f446ze_zio_left.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Zio connectors (top left)
+.. image:: img/nucleo_f446ze_zio_right.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Zio connectors (top right)
+.. image:: img/nucleo_f446ze_morpho_left.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Morpho connectors (top left)
+.. image:: img/nucleo_f446ze_morpho_right.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Morpho connectors (top right)
+
+For mode details please refer to `STM32 Nucleo-144 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_3_TX : PD8
```

Comment:
If you add this rst-class directive and a blank line before the list, it will display the list in multiple columns instead of a long list that runs down the page:
```
.. rst-class:: rst-columns

- UART_3_TX : PD8
- UART_3_RX : PD9
  etc.
```

Reply:
Noted. I've never worked with the Sphinx documentation before, so thanks for the tip.